styleguide/derek/pattern-components/headers/headers.js

```diff
@@ -0,0 +1,44 @@
+Zoolander.Banners = (() => {
+  function interiorBanner() {
+    const $textContainer = $('.refreshBanner-textContainer');
+    const $leftSlant = $('.refreshBanner-line.left-top');
+    const $mainContainer = $('.refreshBanner-container');
+    const $mainContainerLeft = $mainContainer.offset().left;
```

Comment:
the `$` in front of a variable implies you're working with a jQuery object. While true in the three of the cases above this, in this particular case it is not true.  I'd remove the `$` on any variables that aren't jQuery objects to provide better clarity. This applies to all variables in this file.